.claude/scripts/polarion_client.py

```
@@ -0,0 +1,395 @@
+                self.soap_url,
+                data=soap_request,
+                headers=headers,
+                verify=self.verify_ssl,
+                timeout=30
+            )
+
+            if response.status_code == 200:
+                # Parse SOAP response to extract test steps
+                import xml.etree.ElementTree as ET
+
+                root = ET.fromstring(response.text)
```

Ruff (0.15.15)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)